### PR TITLE
Feature/patro23 46 fluent validation

### PR DESCRIPTION
**Pull request by PiotrekkP**

### CODE REVIEW

File: src/modules/example/api/Controllers/ExampleController.cs [hunk 1]
```diff
@@ -36,8 +45,15 @@ public ExampleController(ICommandBus commandBus, IQueryBus queryBus)
...
-		var pagedList = await this.queryBus.Query<GetExamples, PagedList<ExampleInfo>>(new GetExamples());
-		return this.Ok(pagedList);
+		var validator = this.getExamplesValidator.Validate(request);
+		if (validator.IsValid)
+		{
+			var pagedList = await this.queryBus.Query<GetExamples, PagedList<ExampleInfo>>(request);
+			return this.Ok(pagedList);
+		}
+
+		string errorResponse = new ErrorResponse(" ", " ", " ", validator.Errors).ToJson();
```
Comment (mateuszmuszynski) [inline src/modules/example/api/Controllers/ExampleController.cs:55]: Correct me if I'm wrong, but I think this code will never be called as you registered automatic validation in ExampleModule.

Comment (mateuszmuszynski) [inline src/modules/example/api/Controllers/ExampleController.cs:55]: Why you pass empty strings here?

File: src/modules/example/application/Example/ErrorResponse.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+			this.errors = errors;
+		}
+
+		/// <summary>
+		/// Convert error object to JSON.
+		/// </summary>
+		/// <returns>Error response in json format.</returns>
+		public string ToJson()
+		{
+			string json = JsonSerializer.Serialize(this);
```
Comment (mateuszmuszynski) [inline src/modules/example/application/Example/ErrorResponse.cs:44]: I doubt it will work. It will not serialize private fields.

**Review by mateuszmuszynski — CHANGES_REQUESTED** [review]

**Review by mateuszgiedzinski — CHANGES_REQUESTED** [review]

File: src/modules/example/api/Controllers/ExampleController.cs (hunk 1 above)
Comment (PiotrekkP) [reply src/modules/example/api/Controllers/ExampleController.cs:55]: I wasn't sure what exactly should i pass here, so i leave these "empty not empty" strings like this so we can discuss it. But as i can see even the way i am trying to create this error response isn't the best one.

**Review by mateuszmuszynski — APPROVED** [review]

**Review by mateuszgiedzinski — APPROVED** [review]

**Pull request merged**